ads/aja.js

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
```

Comment:
This is fine, don't need to change the year for existing file, you can revert the change here.

Reply:
We will fix it on weekdays

Reply:
fix it!